apps/console/src/features/applications/components/forms/general-details-form.tsx

```diff
@@ -207,7 +210,32 @@ export const GeneralDetailsForm: FunctionComponent<GeneralDetailsFormPopsInterfa
         }
     };
 
+    const {
+        data: myAccountStatus,
+        isLoading: isMyAccountStatusLoading,
+        error: myAccountStatusFetchRequestError,
+        mutate: mutateMyAccountStatusFetchRequest
+    } = getMyAccountStatus();
+
+    /**
+     * Sets the initial spinner.
+     * TODO: Remove this once the loaders are finalized.
+     */
+    useEffect(() => {
+        let status: boolean = AppConstants.DEFAULT_MY_ACCOUNT_STATUS;
+        if ( myAccountStatus ) {
+            const enableProperty = myAccountStatus["value"];
+            if ( enableProperty && enableProperty == "false" ) {
```

Comment:
Lets use `===` to prevent type Coercion.
```suggestion
            if ( enableProperty && enableProperty === "false" ) {
```